code/modules/maps/planet_types/lore/uueoaesa.dm

```diff
@@ -158,7 +158,7 @@
 			surface_color = "#faeac5"
 			//Wasteland ruins
-			//ruin_type_whitelist = list(/datum/map_template/ruin/exoplanet/moghes_guwandi, /datum/map_template/ruin/exoplanet/moghes_gawgaryn_bikers, /datum/map_template/ruin/exoplanet/moghes_kataphract_wasteland, /datum/map_template/ruin/exoplanet/moghes_wasteland_dorviza, /datum/map_template/ruin/exoplanet/moghes_wasteland_ozeuoi, /datum/map_template/ruin/exoplanet/moghes_wasteland_vihnmes, /datum/map_template/ruin/exoplanet/moghes_wasteland_village, /datum/map_template/ruin/exoplanet/moghes_wasteland_izweski, /datum/map_template/ruin/exoplanet/moghes_siakh, /datum/map_template/ruin/exoplanet/moghes_queendom, /datum/map_template/ruin/exoplanet/moghes_wasteland_klax, /datum/map_template/ruin/exoplanet/moghes_wasteland_reclaimer, /datum/map_template/ruin/exoplanet/moghes_wasteland_mikuetz, /datum/map_template/ruin/exoplanet/moghes_wasteland_crater, /datum/map_template/ruin/exoplanet/moghes_wasteland_oasis, /datum/map_template/ruin/exoplanet/moghes_wasteland_battlefield, /datum/map_template/ruin/exoplanet/moghes_ruined_base, /datum/map_template/ruin/exoplanet/moghes_wasteland_tomb, /datum/map_template/ruin/exoplanet/moghes_wasteland_bomb, /datum/map_template/ruin/exoplanet/moghes_wasteland_crash, /datum/map_template/ruin/exoplanet/moghes_wasteland_priests, /datum/map_template/ruin/exoplanet/moghes_dead_guwandi, /datum/map_template/ruin/exoplanet/moghes_gawgaryn_riders, /datum/map_template/ruin/exoplanet/moghes_wasteland_tyrant)
+			ruin_type_whitelist = list(/datum/map_template/ruin/exoplanet/moghes_guwandi, /datum/map_template/ruin/exoplanet/moghes_gawgaryn_bikers, /datum/map_template/ruin/exoplanet/moghes_kataphract_wasteland, /datum/map_template/ruin/exoplanet/moghes_wasteland_dorviza, /datum/map_template/ruin/exoplanet/moghes_wasteland_ozeuoi, /datum/map_template/ruin/exoplanet/moghes_wasteland_vihnmes, /datum/map_template/ruin/exoplanet/moghes_wasteland_village, /datum/map_template/ruin/exoplanet/moghes_wasteland_izweski, /datum/map_template/ruin/exoplanet/moghes_siakh, /datum/map_template/ruin/exoplanet/moghes_queendom, /datum/map_template/ruin/exoplanet/moghes_wasteland_klax, /datum/map_template/ruin/exoplanet/moghes_wasteland_reclaimer, /datum/map_template/ruin/exoplanet/moghes_wasteland_mikuetz, /datum/map_template/ruin/exoplanet/moghes_wasteland_crater, /datum/map_template/ruin/exoplanet/moghes_wasteland_oasis, /datum/map_template/ruin/exoplanet/moghes_wasteland_battlefield, /datum/map_template/ruin/exoplanet/moghes_ruined_base, /datum/map_template/ruin/exoplanet/moghes_wasteland_tomb, /datum/map_template/ruin/exoplanet/moghes_wasteland_bomb, /datum/map_template/ruin/exoplanet/moghes_wasteland_crash, /datum/map_template/ruin/exoplanet/moghes_wasteland_priests, /datum/map_template/ruin/exoplanet/moghes_dead_guwandi, /datum/map_template/ruin/exoplanet/moghes_gawgaryn_riders, /datum/map_template/ruin/exoplanet/moghes_wasteland_tyrant)
```

Comment:
each of the ruins should be in its own line of code and not all in one
instead of `= list(a, b, c, d)`
should be:
```
= list(
  a,
  b,
  c,
  d,
)
```

Reply:
this makes it easier to read
and also means smaller diffs if someone adds/removes/changes just one ruin